src/GafferUIModule/PathListingWidgetBinding.cpp

```diff
@@ -522,16 +562,28 @@ class PathModel : public QAbstractItemModel
 		std::vector<QModelIndex> indicesForPaths( const IECore::PathMatcher &paths )
 		{
 			std::vector<QModelIndex> result;
-			const Path *rootPath = m_rootItem->path();
-			if( !rootPath )
+			if( !m_rootPath )
 			{
 				return result;
 			}
 
-			indicesForPathsWalk( m_rootItem.get(), QModelIndex(), paths, result );
+			indicesForPathsWalk( m_rootItem.get(), Path::Names(), QModelIndex(), paths, result );
 			return result;
 		}
 
+		void waitForPendingUpdates()
+		{
+			if( m_updateScheduled )
```

Comment:
is this necessary ? `startUpdate()` checks m_updateScheduled itself.

Reply:
Good point - addressed in https://github.com/GafferHQ/gaffer/pull/4391/commits/a5f3cb1fe14b09bb65fc5a222b40b7d15c553381.